src/store/reducer.js

```diff
@@ -0,0 +1,115 @@
+import { combineReducers } from "redux";
+import { initCells } from "../Controller/Control";
+import { CHANGE_MUSIC, CHANGE_SOUND, CHANGE_THEME, CHANGE_COLOR, NEW_FIELD, SET_SIZE , UPDATE_FIELD, CHANGE_SIZE, INCREMENT, RESET_SCORE, MUSIC_VOLUME, SOUND_VOLUME, FETCH_USERDATA, SIGN_OUT, FETCH_LEADERBOARD, LOADER_DEACTIVATE, LOADER_ACTIVATE } from "./types";
+
+
```

Comment:
It's better to use initial states in every reducer.
Also (not critical) split reducers to separate files